CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Added 75 more blocks to the default atheneum list.
 - Transportalizer particle
 
+### Changed
+
+- Updated textures for Rainbow Logs, Planks, Leaves and Saplings
+
+### Removed
+
+- Removed unused blockstates and models for the Rainbow Sapling
```

Comment:
Calling them unused is a bit misleading since they weren't unused before this update. Perhaps phrase it as a change to a simpler growth process without colored state properties?